frontends/mit-open/package.json

```diff
@@ -53,7 +53,7 @@
   },
   "dependencies": {
     "@ebay/nice-modal-react": "^1.2.13",
-    "@mitodl/course-search-utils": "^3.0.9",
+    "@mitodl/course-search-utils": "https://github.com/mitodl/course-search-utils.git#a4efeaa70c4c4677d96d5a1a89e1e4b0eb64ae31",
```

Comment:
Prior to merging, this will be updated to a published version once 
- https://github.com/mitodl/course-search-utils/pull/106

is reviewed.